openbb_terminal/stocks/tradinghours/data/bursa.py

```diff
@@ -0,0 +1,1093 @@
+from datetime import time
```

Comment:
I would replace this python file by a `yaml` (or `.ini` or `json`...) file and load it with a function.

Since it's a data file and not code.

Plus this will allow allow not having to ignore this file from the `codespell` check.

Reply:
done